Kitodo/src/main/java/de/sub/goobi/helper/ldap/LdapUser.java

```diff
@@ -77,6 +76,7 @@ public LdapUser() {
      */
     public void configure(User inUser, String inPassword, String inUidNumber)
             throws NamingException, NoSuchAlgorithmException, IOException, InterruptedException {
+        JDKMessageDigest.MD4 digester = new JDKMessageDigest.MD4();
```

Comment:
`JDKMessageDigest` could be removed or did I miss something why it should used in this way?